contracts/package.json

```diff
@@ -12,6 +12,7 @@
     "check-ts": "tsc --noEmit",
     "lint": "eslint . --ext .ts",
     "test": "hardhat test",
+    "test-clients": "hardhat test ./clients-tests/*",
```

Comment:
Not 100% sure this new directory is the best home for these tests so open to suggestions. Didn't put them in `./clients` as they require hardhat, and the client module isn't a hardhat project.

Would also like to get thoughts on integrating these as automated tests, but unsure on how to do that.

Reply:
Given the dependency the clients have with the contracts, we probably should just setup integration style tests for them in addition to the current unit tests.

For automation, we could split the client GH action [test](https://github.com/web3well/bls-wallet/blob/main/.github/workflows/clients.yml#L26-L32) step into two or just include integration tests in it. See also the gas measurement PR where I [added a local contract deploy action](https://github.com/web3well/bls-wallet/pull/341/files#diff-21998d19187400c87c1b4d6bb4ccbe519f3c9f5fcff783d76f392d7640449de4) that could be used here as well.

Reply:
As discussed on a call. Will address this in separate issue
[#414 Setup integration tests for BlsProvider & BlsSigner](https://github.com/web3well/bls-wallet/issues/414)